src/main/pg/pg_ids.h

```diff
@@ -150,7 +150,8 @@
 #define PG_PULLUP_CONFIG 551
 #define PG_PULLDOWN_CONFIG 552
 #define PG_MODE_ACTIVATION_CONFIG 553
-#define PG_BETAFLIGHT_END 553
+#define PG_RX_MSP_OVERRIDE_CONFIG 554
```

Comment:
Not sure this needs a parameter group of its own, what is the reason to not add this to an existing one, like `PG_RX_CONFIG`?

Reply:
No reason, can add to an existing one instead